inst/sql/sql_server/field_cdm_field.sql

```diff
@@ -7,17 +7,21 @@ Parameters used in this template:
 cdmDatabaseSchema = @cdmDatabaseSchema
 cdmTableName = @cdmTableName
 cdmFieldName = @cdmFieldName
+isPostgresql = @isPostgresql
```

Comment:
Is this not handled by SqlRender? Unclear as why there is DB specific instruction.  Might just need to understand.

Reply:
Misunderstanding on my part -- it's not that PG was an issue, it was the casts added in the CSV. I've removed this hack.